.github/workflows/pr-title.yaml

```
@@ -0,0 +1,24 @@
+    steps:
+      - name: Checkout
+        uses: actions/checkout@df4cb1c069e1874edd31b4311f1884172cec0e10 # v6.0.3
+        with:
+          fetch-depth: 0
+      - uses: './.github/actions/setup-environment'
+      - name: Install
+        shell: bash
+        run: npm ci
+      - name: Validate PR title with commitlint
+        shell: bash
+        run: echo "${{ github.event.pull_request.title }}" | npx commitlint --verbose
```

actionlint (1.7.12)

[error] "github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)